bears/c_languages/codeclone_detection/ClangFunctionDifferenceBear.py

```diff
@@ -46,6 +48,29 @@
 member_accessed"""))
 
 
+@generate_repr(('id', hex),
+               'origin',
+               'differences',
+               'count_matrices',
+               'message')
+@generate_ordering('origin',
+                   'differences',
+                   'count_matrices',
+                   'message')
+class ClangFunctionDifferenceResult(HiddenResult):
+
+    @enforce_signature
+    def __init__(self, origin,
+                 differences: list,
+                 count_matrices: dict):
+        HiddenResult.__init__(origin,
+                              [differences, count_matrices],
+                              ('ClangFunctionDifferenceResult: differences[%d]'
```

Comment:
is this necessary?

if not, remove, as we dont want to enhance the deprecated '.contents'

Reply:
@jayvdb Well, yeah, it's not too necessary, it's just for the `message` argument, I can remove it..

But, did you mean to point to line `67`? that's the `contents` argument

Reply:
Sorry i meant '.message' which was previously unused and isnt very useful for this bear.